### DISCUSSION TITLE
Unclickable right sidebar with years in profile of user

### DISCUSSION THREAD

**Discussion by borwi4ok** [community/community#73320] [Accessibility] [answered]
### Select Topic Area

Bug

### Body

Testcase:
1. open profile with year of activity starts from 2009
2. scroll to the bottom of page
3. option 2009 is not clickable because basement overflow this option
<img width="1274" alt="Снимок экрана 2023-11-03 в 15 38 02" src="https://github.com/community/community/assets/33358994/7e603e47-3f1e-4c07-b33d-d3567d10380e">

**Comment by ghostinhershell** [comment]
Hi @borwi4ok, 

Thanks so much for reaching out here. I tested this out on my end by finding a profile with a similar contribution history and I do see that the dates do overflow in a way where it passes the footer line on the page. However, every year on that list is clickable on my end. 

What browser are you currently using to view this page?

**Reply by borwi4ok** [reply, marked as answer]
@ghostinhershell sorry for long time reply, I see that basement is changed and now basemenet is not overflowing years options, thx for fix

**Reply by ghostinhershell** [reply]
No problem, thanks so much for confirming!

**Comment by rahmatez** [comment]
![image](https://github.com/user-attachments/assets/294b48c9-e80d-4404-b718-94854131a63a)
I also experienced the same thing, how to fix it? is there a solution?
